Cartography/ViewProxy.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ViewProxy.swift
+//  Cartography-iOS
+//
+//  Created by Vitor Travain on 10/10/17.
+//  Copyright © 2017 Robert Böhnke. All rights reserved.
+//
+
+public final class ViewProxy: SupportsPositioningLayoutProxy {
+    public var context: Context
+    public var element: AnyObject
+
+    init(context: Context, view: View) {
+        self.context = context
+        self.element = view
+    }
+
+    public var superview: ViewProxy? {
+        guard let view = element as? View else {
+            fatalError("This wasn't supposed to happen")
+        }
+
+        guard let superview = view.superview else {
+            return nil
+        }
+
+        return ViewProxy(context: context, view: superview)
+    }
+}
```

Comment:
I was thinking if it would be a good idea to have a computed variable for `safeAreaLayoutGuide` in case the platform is iOS.

Reply:
Where exactly this computed variable would be placed though?

Reply:
Below superview in the `ViewProxy` class

Reply:
Why though? Convenience?

Reply:
Yep

Reply:
It's been done. Please review again @gsampaio